infrastructure/template.yaml

```diff
@@ -63,6 +63,18 @@ Mappings:
       integration: arn:aws:secretsmanager:eu-west-2:216552277552:secret:DynatraceNonProductionVariables
       production: arn:aws:secretsmanager:eu-west-2:216552277552:secret:DynatraceProductionVariables
 
+  Audit:
+    EventName:
+      ResponseReceived: IPV_CHECK_HMRC_CRI_RESPONSE_RECEIVED
+      RequestSent: IPV_CHECK_HMRC_CRI_REQUEST_SENT
```

Comment:
We should use the AuditEventPrefix - see https://github.com/govuk-one-login/ipv-cri-check-hmrc-api/pull/153/commits/ad66efd5280519bc2167e207825bd59fe0ec56b7